### PR TITLE
Fix variableScope

### PR DESCRIPTION
**Pull request by egecetin**
Fix cppcheck warning variableScope

### CODE REVIEW

File: Packet++/src/HttpLayer.cpp [hunk 1]
```diff
@@ -793,9 +793,9 @@ int HttpResponseFirstLine::getStatusCodeAsInt() const
 std::string HttpResponseFirstLine::getStatusCodeString() const
 {
 	std::string result;
-	int statusStringOffset = 13;
 	if (m_StatusCode != HttpResponseLayer::HttpStatusCodeUnknown)
 	{
+		int statusStringOffset = 13;
```
Comment (seladb) [inline Packet++/src/HttpLayer.cpp:798]: I'm not sure I understand this change. Maybe we should turn this variable into a `const`?

File: Packet++/src/SipLayer.cpp [hunk 2]
```diff
@@ -753,9 +753,9 @@ int SipResponseFirstLine::getStatusCodeAsInt() const
 std::string SipResponseFirstLine::getStatusCodeString() const
 {
 	std::string result;
-	int statusStringOffset = 12;
 	if (m_StatusCode != SipResponseLayer::SipStatusCodeUnknown)
 	{
+		int statusStringOffset = 12;
```
Comment (seladb) [inline Packet++/src/SipLayer.cpp:758]: ditto

File: Packet++/src/HttpLayer.cpp (hunk 1 above)
Comment (seladb) [reply Packet++/src/HttpLayer.cpp]: What I meant is that now we can move this back to line 796 and `cppcheck` should not give a warning?

Comment (egecetin) [reply Packet++/src/HttpLayer.cpp]: Oh sorry I understand now fixing

**Review by seladb — APPROVED** [review]

**Pull request merged**